### ISSUE TITLE
--ignore-scripts ignores lifecycle scripts in dependencies?

### ISSUE THREAD

**Issue by olalonde** [yarnpkg/yarn#3230]
**Do you want to request a *feature* or report a *bug*?**

Both

**What is the current behavior?**

`yarn install --ignore-scripts` ignores lifecycle scripts of dependencies.

**What is the expected behavior?**

`yarn install --ignore-scripts` should only ignore lifecycle scripts of package being installed. I am not sure if this is intentional but if it is, having a separate flag for just ignoring the "postinstall" of the local package would be very useful (esp. in Dockerfile).

**Please mention your node.js, yarn and operating system version.**

node v7
yarn v0.22.0

**Comment by bestander** [comment]
Yeah, afaik there is no way to ignore only script of top level package.

**Comment by BYK** [comment]
I think this is by design so I'm marking this as a feature request. Also see https://github.com/yarnpkg/rfcs/pull/76 which is related.
